docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+security of the network depends on the amount of capital locked on the chain: the more the capital
+locked, the lower the chance of an attack on the network, as the attacker
+needs to incur a heavy loss to orchestrate a successful attack (more on this later on). The process of
+locking tokens on the chain is called `staking`.
+
+Similar to the miners in PoW networks, PoS networks have `validators`, but they do not have to compete with each other to solve mathematical puzzles.
+They are instead pre-selected to produce the blocks based on the stake backing them.
+Token holders can lock funds on the chain and for doing so, they are getting `staking rewards`. There is
+thus an economic incentive for token holders to become active participants who contribute to the
+security and economic stability of the network. PoS networks in general are therefore more inclusive
+than PoW networks, as participants do not need to have either technical knowledge about
+blockchain technology or experience in running mining equipment. 
+
+PoS ensures that everybody participating in the staking process has
+"skin in the game" and thus can be held accountable. In case of misbehavior, participants in the
+staking process can be punished or `slashed`, and depending on the gravity of the situation, their
+stake can be partly or fully confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.
+
+## Nominated Proof-of-Stake Overview
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements [Nominated Proof-of-Stake (NPoS)](learn-consensus.md/#nominated-proof-of-stake),
+a relatively novel and sophisticated mechanism to select the validators who are allowed to participate in 
+its [consensus](learn-consensus.md) protocol. The NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+holders to participate as nominators. NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
+holders to participate as `nominators`. 
+
 Nominators may back up to
 {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
-validators as trusted validator candidates. Both validators and nominators lock their tokens as
-collateral and receive staking rewards.
+validators as trusted validator candidates, and the network will automatically distribute the stake among validators in
+an even manner. Also, in NPoS the stake of both nominators and validators can be slashed.  For an in-depth review of NPoS see
+[this](https://research.web3.foundation/en/latest/polkadot/NPoS/index.html) research article.
+
+### Nominating Validators
+
+The action of nominating consists of a) locking or
```

Comment:
The "lock", "bond" and "stake" terms tend to be used interchangeably. Maybe simplify to:

Nominating on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} requires 2 actions:

 a) Locking tokens on-chain.
 b) Selecting a set of validators, to whom these locked tokens will automatically be allocated to. 

How many tokens you lock up is completely up to you - as are the validators you wish to select.

_Locking tokens is also known as **bonding**.  You can also refer to your locked tokens as your **bonded** tokens, or **staked** tokens. Likewise, selecting validators is also known as **backing** or **nominating** validators. These terms are used interchangeably by the community._

From now on locked tokens will be referred to as **bonded** tokens.

So, once the previous 2 steps are completed and you are nominating, your bonded tokens could be allocated to one or more of your selected validators, and this happens _every time the active **validator set** changes_. 

This validator set is updated every **era** on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}. On Polkadot, an era is 1 day.

Unlike other staking systems, Polkadot automatically chooses which of your selected validators will be backed by your bonded tokens. Selecting a group of validators increases your chances of consistently backing at least one who is active. This results in your bonded tokens being allocated to validators more often, which means more network security and more rewards.

This is in stark contrast to other staking systems that only allow you to back one validator; if that validator is not active, you as a staker will also not be. 

Polkadot's nomination model solves this.